demo/build.gradle

```diff
@@ -2,11 +2,13 @@ apply plugin: 'com.android.application'
 
 dependencies {
 
-    compile project(':core')
-    compile project(':merlin-rxjava')
-    compile project(':merlin-rxjava2')
+    implementation project(':core')
+    implementation project(':merlin-rxjava')
+    implementation project(':merlin-rxjava2')
+    implementation libraries.app.rxJava
```

Comment:
Using `implementation` does not expose RxJava to the users of the library directly. That's why we need to explicitly add RxJava here. This is better because users know what's happening.